<?xml version="1.0" encoding="UTF-8"?><record_update table="sys_script_include">
    <sys_script_include action="INSERT_OR_UPDATE">
        <access>public</access>
        <active>true</active>
        <api_name>x_snc_crisis.LogError</api_name>
        <caller_access>1</caller_access>
        <client_callable>false</client_callable>
        <description>Produces an error log output in the application log.</description>
        <mobile_callable>false</mobile_callable>
        <name>LogError</name>
        <sandbox_callable>false</sandbox_callable>
        <script><![CDATA[function LogError(vPayload)
{
	gs.error(x_snc_devtools.PrepareLogError(vPayload));
}]]></script>
        <sys_class_name>sys_script_include</sys_class_name>
        <sys_created_by>admin</sys_created_by>
        <sys_created_on>2023-08-19 12:07:03</sys_created_on>
        <sys_id>ba54b63497387150a6a07767f053afee</sys_id>
        <sys_mod_count>0</sys_mod_count>
        <sys_name>LogError</sys_name>
        <sys_package display_value="Crisis" source="x_snc_crisis">d659aacfdbafc114bcced03cd396198b</sys_package>
        <sys_policy/>
        <sys_scope display_value="Crisis">d659aacfdbafc114bcced03cd396198b</sys_scope>
        <sys_update_name>sys_script_include_ba54b63497387150a6a07767f053afee</sys_update_name>
        <sys_updated_by>admin</sys_updated_by>
        <sys_updated_on>2023-08-19 12:07:03</sys_updated_on>
    </sys_script_include>
</record_update>
